src/lua/help_en_US.lua

```diff
@@ -1,4 +1,4 @@
---- help (en_US)
+--- help (en_US) 2.x
```

Comment:
Don't see a reason to hardcode a version here, because it would be like a yer in copyright notes: often slightly out of date.

Reply:
Elena says it helps to don't confuse between 2.x / 1.10 tutorials when working on them. So it worth to do.